.github/workflows/changelog.yml

```diff
@@ -6,11 +6,32 @@ on:
 permissions: "read-all"
```

Comment:
I think we only need "pull-requests: read" permission here.

Reply:
FINC